.github/workflows/build.yml

```diff
@@ -57,3 +57,8 @@ jobs:
         uses: actions/upload-artifact@v2.2.4
         with:
           path: export.json
+
+      - name: Upload import.gexf
+        uses: actions/upload-artifact@v2.2.4
+        with:
+          path: mathlib_docs/docs/import.gexf
```

Comment:
```suggestion
```
This was just needed for me to get hold of a copy; in future it should end up uploaded alongside the `decl.bmp`.